[2025-07-29T15:55:28.892+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-29T15:55:30.475+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sequential_scraping_pipeline.process_urls_sequentially manual__2025-07-29T14:22:04.997357+00:00 [queued]>
[2025-07-29T15:55:30.859+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sequential_scraping_pipeline.process_urls_sequentially manual__2025-07-29T14:22:04.997357+00:00 [queued]>
[2025-07-29T15:55:30.861+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 1
[2025-07-29T15:55:31.360+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): process_urls_sequentially> on 2025-07-29 14:22:04.997357+00:00
[2025-07-29T15:55:31.413+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=200) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-29T15:55:31.419+0000] {standard_task_runner.py:64} INFO - Started process 219 to run task
[2025-07-29T15:55:31.421+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'sequential_scraping_pipeline', 'process_urls_sequentially', 'manual__2025-07-29T14:22:04.997357+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/sequential_pipeline_dag.py', '--cfg-path', '/tmp/tmpdk7ympdz']
[2025-07-29T15:55:31.423+0000] {standard_task_runner.py:91} INFO - Job 154: Subtask process_urls_sequentially
[2025-07-29T15:55:32.202+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-29T15:55:34.288+0000] {task_command.py:426} INFO - Running <TaskInstance: sequential_scraping_pipeline.process_urls_sequentially manual__2025-07-29T14:22:04.997357+00:00 [running]> on host 64b54ead5d4a
[2025-07-29T15:55:35.217+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2025-07-29T15:55:35.313+0000] {process_utils.py:132} INFO - Sending 15 to group 219. PIDs of all processes in the group: [219]
[2025-07-29T15:55:35.322+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 219
[2025-07-29T15:55:35.537+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=219, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='15:55:31') (219) terminated with exit code -15
[2025-07-29T15:55:35.545+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 143
[2025-07-29T15:55:37.932+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
